kiwi/firmware.py

```diff
@@ -160,7 +164,8 @@ def get_legacy_bios_partition_size(self):
         Size of legacy bios_grub partition if legacy BIOS mode is
         required. Returns 0 if no such partition is needed
 
-        :return: mbsize
+        :return: mbsize value
```

Comment:
"... as integer"?

Reply:
isn't that clear by the line below saying :rtype: int ?

Reply:
Oh, right, sorry.